docs/guides/integration-options-publisher-web.md

```diff
@@ -0,0 +1,272 @@
+
+To choose your implementation and get started, follow these steps:
+
+1. Review the summary of options to generate a UID2 token:
+   - [Client-Side Integration Options](#client-side-integration-options)
+   - [Server-Side Integration Options](#server-side-integration-options)
+1. Review the options to [refresh the UID2 token](#refresh-the-uid2-token).
+1. Review the options to [pass the token into the bid stream](#pass-the-uid2-token-into-the-bid-stream).
+1. Choose the option that's best for you, and then click through to the implementation documentation.
+
+## Generate the UID2 Token
+
+There are two main paths that you can choose to generate a UID2 token: you can choose to initiate the UID2 token generate request on the client side (in the user's browser) or on the server side.
+
+Each option has different advantages. Client-side integration is easy and fast; integration using Prebid.js 8.21.0 or later is the easiest and fastest integration option.
+
+:::note
+For all integration options, you can choose to store the UID2 token in local storage or cookie storage.
+:::
+
+### Client-Side Integration Options
+
+Generating the UID2 token on the client side has the following advantages:
+
+- The code runs on the client side, on the consumer's web page, and no server-side coding is required.
+- There is a Prebid integration that handles all functions for you&#8212;token generation, token refresh, and passing the token into the bid stream. If you can use Prebid 8.21.0 or later, this is the simplest and easiest implementation option.
+
+If you choose a client-side integration, you'll need to provide a list of your top-level domains, for security purposes, as part of account setup. For details, see [Client-Side Implementation for Publishers](../getting-started/gs-account-setup.md#client-side-implementation-for-publishers) on the Account Setup page.
+
+The following table summarizes the options for publishers who want to generate the UID2 token on the client side, via the web page, with corresponding documentation resources.
+
+| Option | Documentation |
+| :--- | :--- |
+| Prebid.js client-side implementation (8.21.0 or later) | [UID2 Client-Side Integration Guide for Prebid.js](integration-prebid-client-side.md) |
+| UID2 SDK for JavaScript, client-side implementation | [JavaScript Express Integration Guide](publisher-client-side.md) |
+
+### Server-Side Integration Options
+
+Generating the UID2 token on the server side has the following advantages:
+
+- You can keep your [directly identifying information (DII)](../ref-info/glossary-uid.md#gl-dii) entirely on the server side.
+- If your development resources are back-end developers, you might prefer a server-side integration.
+- For server-side Prebid integration, there is no requirement to update to the latest Prebid version. If you're using Prebid, and have a constraint with regard to version, choose server-side integration.
```

Comment:
the way i read this is "if i do server side integration with prebid.js, i can use any version of prebid.js", which isn't true and we specify 7.53.0 or later in other sections. 

just raising this and up to you if you wanna rephrase it.

Reply:
@sunnywu need to discuss to clarify.

Reply:
Good point, thx. Changed it to:
For server-side Prebid integration, there is no requirement to update to the latest Prebid version, as long as your version is 7.53.0 or later. If you're using Prebid, and have a constraint with regard to version, choose server-side integration.